.github/workflows/build-image-base.yml

```diff
@@ -22,6 +22,10 @@ on:
         default: linux/amd64
         type: string
         description: "Platforms to build for"
+      test_platform:
+        default: linux/amd64
+        type: string
+        description: "Platform to use for acceptance tests"
```

Comment:
In each image yaml file there is this line - `platforms: linux/amd64,linux/arm64` so the image is built for each of those platforms. Without the **test_platform** spec the workflow step for the acceptance tests was failing because there is no facility for running tests on arm64 architecture. Don't forget that these tests have not been running for over a year and consequently a lot of changes have been added to this repo without those changes passing the acceptance test steps.

Perhaps the better way forward is for me to look into creating a container for testing the arm64 version of the image.